### DISCUSSION TITLE
native white balance is the best for the most manual control and the least processed image?

### DISCUSSION THREAD

**Discussion by temtekmedia** [gopro/labs#1134] [General]
so when I set white balance to native, it's using the full sensor gamut, so wide=1 is not needed with the native white balance. 

However I'm curious, is the native white balance something like static 5500k or does it mean that the white balance isn't baked into the image like with raw videos from cine cameras? I'm just wondering whats the absolute best way to approach this to gain maximum manual control and the least processed image for color grading.

**Comment by Deadcode85** [comment]
Wide=1 and manually set WB is your best aproach to get maximum control. Native is the old Wide gamut option since Hero3+(?) which was good for a long time, but if you had to push the WB in 3200K environment, to image falled apart.Maybe it's better now with 10 bit codec, but still, proper WB is always better starting point.
Native was around 7000K WB.

**Reply by temtekmedia** [reply]
Ahh, Interesting.. The creator of the GP-tune plugin/transform xtremestuff told me that the native WB would be the best way for maximum control, and that wide=1 would be best for auto white balance, so I got a bit confused since I didnt understand the connection between white balance and color space, I've gotta do some more grading tests but so far I got very good results with native WB & wide=1 and also 5500k WB & wide=1

edit: Here It's explained on the official labs documents aswell so you're right, native WB is pretty much same as wide=1 but with baked in WB @ ~7000k    https://gopro.github.io/labs/control/extensions/
